pkg/storage/kube/project.go

```diff
@@ -174,6 +175,7 @@ func NewProjectFromSecret(secret *v1.Secret, namespace string) (*brigade.Project
 	proj.Kubernetes.BuildStorageSize = def(sv.String("buildStorageSize"), "50Mi")
 	proj.Kubernetes.BuildStorageClass = sv.String("kubernetes.buildStorageClass")
 	proj.Kubernetes.CacheStorageClass = sv.String("kubernetes.cacheStorageClass")
+	proj.Kubernetes.ServiceAccount = sv.String("serviceAccount")
```

Comment:
This is going to break backwards compatibility, not sure if we want to prevent issues by adding some sort of `StringOrDefault` to `SecretValues`. Thoughts?

Reply:
Oh actually nevermind.